tests/datasets/test_mmflood.py

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+from itertools import product
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import pytest
+import torch
+import torch.nn as nn
+from _pytest.fixtures import SubRequest
+from pytest import MonkeyPatch
+from rasterio.crs import CRS
+
+from torchgeo.datasets import (
+    BoundingBox,
+    DatasetNotFoundError,
+    IntersectionDataset,
+    MMFlood,
+    UnionDataset,
+)
+
+
+class TestMMFlood:
+    @pytest.fixture(
+        params=product([True, False], [True, False], ['train', 'val', 'test'])
+    )
+    def dataset(
+        self, monkeypatch: MonkeyPatch, tmp_path: Path, request: SubRequest
+    ) -> MMFlood:
+        dataset_root = os.path.join('tests', 'data', 'mmflood/')
+        url = os.path.join(dataset_root)
```

Comment:
```suggestion
        url = os.path.join('tests', 'data', 'mmflood') + os.sep
```
Use `os.sep` to make it platform-independent